test-lib-openshift.sh

```diff
@@ -94,6 +94,27 @@ function ct_os_wait_pod_ready() {
   while ! ct_os_check_pod_readiness "${pod_prefix}" "true" ; do
     echo -n "."
     [ ${SECONDS} -gt ${timeout} ] && echo " FAIL" && return 1
+    SECONDS=$((SECONDS+3))
+    sleep 3
+  done
+  echo " DONE"
+}
+
+# ct_os_wait_rc_ready [pod_prefix, timeout]
+# --------------------
```

Comment:
Nit, with square brackets it usually means the arguement is optional. This header is more readable:
```
# ct_os_wait_rc_ready POD_PREFIX TIMEOUT
# --------------------------------------
```

Reply:
Makes sense. Other function docs updated as well.